l10n/en/firefox/sticky-promo.ftl

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/*
+### Global to all /firefox pages
+
+# Meet our family of products
+firefox-sticky-promo-meet-our-family-of = Meet our family of products
```

Comment:
hmm... well if that string is in a global file you can reference the string ID in the template, rather than doing it like this (which I expect might cause other problems, if it works at all). And if it's not in a global file you'd need to reference that file from every page that wants to use it, which can make a mess with file activation.

So even though this string exists elsewhere I think it's probably still best to treat it as new.